core/vibe/core/net.d

```diff
@@ -434,6 +434,24 @@ interface UDPConnection {
 	ubyte[] recv(ubyte[] buf = null, NetworkAddress* peer_address = null);
 	/// ditto
 	ubyte[] recv(Duration timeout, ubyte[] buf = null, NetworkAddress* peer_address = null);
+
+	version (VibeLibeventDriver) {
```

Comment:
Instead of making the interface specific to libevent, the other drivers should just have simple stubs with `assert(false, "TODO!");` inside. It would also be good to update the `vibe-core` package accordingly, but I can do that, too.